app/views/partials/_header.html.haml

```diff
@@ -0,0 +1,9 @@
+#header.flex.items-center.justify-between.flex-wrap.bg-grey-darkest.px-16.py-6
```

Comment:
Lets extract these long classes to tailwind.scss

Reply:
Will do